apps/wallet-mobile/src/yoroi-wallets/hooks/index.ts

```diff
@@ -123,6 +124,14 @@ export const useStakingKey = (wallet: YoroiWallet) => {
   return result.data
 }
 
+export const useAddressHex = (wallet: YoroiWallet) => {
+  const result = useQuery([wallet.id, 'addressHex'], () => convertBech32ToHex(wallet.externalAddresses[0]), {
```

Comment:
if it is a SA wallet, it should take from the internal[0], is it for the swap trade deposit?

Reply:
Some endpoints want this format. Not sure if internal/external makes a difference.